influxql.go

```diff
@@ -187,7 +196,13 @@ func (n *InfluxQLNode) runBatchInfluxQL() error {
 				kind = reflect.Float64
 			}
 		} else {
-			k := reflect.TypeOf(b.Points[0].Fields[c.field]).Kind()
+			f, ok := b.Points[0].Fields[c.field]
+			if !ok {
+				n.incrementErrorCount()
+				n.logger.Printf("E! field %s missing from point cannot get type\n", c.field)
```

Comment:
I think you need to stop the timer here as well.

Also, this skips the entire batch if the first point is missing the field. I think that is ok behavior but we should update the log message so that it is clear. Something like `E! field %s missing, skipping batch`